styleguide/components/00-settings/_colors.scss

```diff
@@ -24,7 +24,7 @@ $colors: (
   'white': #fff,
   'blue': #0340c7,
   'teal': #29cfc9,
-  'green': #2f8727, // wcag compliant
+  'green': #41ba36, // wcag compliant
```

Comment:
In de preview staat een andere kleur in de config

Reply:
Fixed